### PR TITLE
Bluesky reply options

### PR DESCRIPTION
**Pull request by aneillans**
Adds the ability to limit who can reply to your BlueSky posts as per the recent update

### CODE REVIEW

File: electron-app/src/server/websites/bluesky/bluesky.service.ts [hunk 1]
```diff
@@ -261,6 +266,12 @@ export class Bluesky extends Website {
       const postId = postResult.uri.slice(postResult.uri.lastIndexOf('/') + 1);
 
       let friendlyUrl = `https://${server}/profile/${handle}/post/${postId}`;
+
+      // After the post has been made, check to see if we need to set a ThreadGate; these are the options to control who can reply to your post, and need additional calls
+      if (data.options.threadgate != "") {
```
Comment (mvdicarlo) [inline electron-app/src/server/websites/bluesky/bluesky.service.ts:271]: nit: != -> !== unless you have an explicit need for the coercion to happen

**Comment by mvdicarlo** [comment]
Was a lint run over this?

File: electron-app/src/server/websites/bluesky/bluesky.service.ts (hunk 1 above)
Comment (aneillans) [reply electron-app/src/server/websites/bluesky/bluesky.service.ts]: Fixed

**Comment by aneillans** [comment]
> Was a lint run over this?

Yup, nothing returned.

**Comment by mvdicarlo** [comment]
merge conflict with the tag pr that was merged unfortunately

**Comment by aneillans** [comment]
> merge conflict with the tag pr that was merged unfortunately

Give me 10 minutes and I'll update :)

**Comment by aneillans** [comment]
Should be good now @mvdicarlo

**Pull request merged**